sql/moz-fx-data-shared-prod/adjust/adjust_kpi_deliverables/view.sql

```diff
@@ -2,5 +2,26 @@ CREATE OR REPLACE VIEW
     `moz-fx-data-shared-prod.adjust.adjust_kpi_deliverables`
 AS
 SELECT
-    *
+    date,
+    app,
+    network,
+    network_token,
+    campaign,
+    campaign_token,
+    adgroup,
+    adgroup_token,
+    creative,
+    creative_token,
+    country,
+    os,
+    device,
+    clicks,
+    installs,
+    limit_ad_tracking_install_rate,
+    click_conversion_rate,
+    impression_conversion_rate,
+    sessions,
+    daus,
+    waus,
+    maus
 FROM `moz-fx-data-shared-prod.adjust_derived.adjust_derived_v1`
```

Comment:
The columns in the list don't match the [`adjust_derived_v1` schema](https://github.com/mozilla/bigquery-etl/blob/446ed0dc2c456dc697fcf79340cefb4ee16fab00/sql/moz-fx-data-shared-prod/adjust_derived/adjust_derived_v1/schema.yaml).

Reply:
It should be the same except for the app_token. I wanted to take that out. Would this cause failures?